include/deal.II/lac/trilinos_solver.h

```diff
@@ -568,6 +568,21 @@ namespace TrilinosWrappers
     virtual ~SolverDirect ();
 
     /**
+     * Initializes the direct solver for the matrix <tt>A</tt> and creates a
+     * factorization for it with the package chosen from the additional
+     * data structure. Note that there is no need for a preconditioner
+     * here and there is no solve performed.
```

Comment:
perhaps the end of the sentence should be changed a bit like `...here and solve() is not called`.